app/templates/_gradle.properties

```diff
@@ -35,7 +35,7 @@ mysql_connector_java_version=5.1.36<% } %>
 h2_version=1.4.188
 <% if (testFrameworks.indexOf('gatling') != -1) { %>
 gatling_version=2.1.7<% } %>
-mapstruct_version=1.0.0.CR1<% if (enableSocialSignIn) { %>
```

Comment:
We have a bug with CR2, we should stick to CR1 (as with the Maven build normally)